File: CS/Bike.cs
public class Bike : Landway
{
    private int engineCC;

    public Bike(int id, string name, string engine, int cc)
        : base(id, name, engine)
    {
        engineCC = cc;
    }

    public override void DisplayDetails()
    {
        base.DisplayDetails();
        Console.WriteLine("Engine CC       : " + engineCC);
    }

    public override void StartJourney()
    {
        Console.WriteLine("Bike Journey Started Successfully...");
    }
}

File: Java/Bike.java
class Bike extends Landway
{
    private int engineCC;

    Bike(int id, String name, String engineType, int engineCC)
    {
        super(id, name, engineType);
        this.engineCC = engineCC;
    }

    public void displayDetails()
    {
        super.displayDetails();
        System.out.println("Engine CC      : " + engineCC + " cc");
    }

    @Override
    void startJourney()
    {
        System.out.println("Bike Journey Started Successfully...");
    }
}